sdk/metrics/src/jmh/java/io/opentelemetry/sdk/metrics/internal/aggregator/HistogramValueGenerator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.aggregator;
+
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.function.DoubleSupplier;
+
+/** Methods of generating values for histogram benchmarks. */
+@SuppressWarnings("ImmutableEnumChecker")
+public enum HistogramValueGenerator {
+  SAME_VALUE(() -> 100.0056),
+  RANDOM_WITHIN_2K(() -> ThreadLocalRandom.current().nextDouble(2000));
```

Comment:
Random JMH tends to be not that useful, especially here it seems like numbers could be vastly different across runs. How about various golden lists of measurements that are all recorded in a single iteration rather than one random value?

Reply:
Two things:

1. The resizing/reallocating aspect of memory is important here.  `MapCounter` performs abysmally in other benchmarks I've targeted, but I can duplicate that in a no-warmup benchmark to show the cost over time.
2. While numbers CAN be vastly different over runs, I've seen pretty stable / repeated performance output here.  This is because the algorithms *should* be resilient to randomness in practice.

I can update this to randomnly generate the values and use the same ones each test, but I'll need to also go update to make sure warmup isn't preserving allocated memory in ways that won't reflect production.